mlflow/models/container/__init__.py

```diff
@@ -128,12 +127,11 @@ def _serve_pyfunc(model):
     # link the log streams to stdout/err so they will be logged to the container logs
     check_call(['ln', '-sf', '/dev/stdout', '/var/log/nginx/access.log'])
     check_call(['ln', '-sf', '/dev/stderr', '/var/log/nginx/error.log'])
-    cpu_count = multiprocessing.cpu_count()
```

Comment:
I'm a little worried about removing this. In particular this code path is also called into by the `mlflow sagemaker build-and-push-container` CLI command, and changing this alters the existing behavior of trying to dynamically set the number of workers based on the number of available CPUs, which is also what an (old) SageMaker example does ([link](https://github.com/awslabs/amazon-sagemaker-examples/blob/65f36146474b1404b28a81c0bb2ed41c7955f15c/advanced_functionality/scikit_bring_your_own/container/decision_trees/serve#L26)). I wonder if we could set `-w` to `multiprocessing.cpu_count()` by default, but allow users to override this by specifying custom gunicorn opts? So our final command on line 134 would look like:

```
cmd = "gunicorn -w {cpu_count} ${GUNICORN_CMD_ARGS} mlflow.models.container.scoring_server.wsgi:app".format(cpu_count=multiprocessing.cpu_count())
```

Of course we should test that overriding `-w` works in the above case^

Reply:
Good suggestion Sid! I verified locally that `-w` can be overridden.